### PR TITLE
Update notepadplusplus to wait on config file

### PR DESCRIPTION
**Pull request by MalwareMechanic**
Also updated the spacing from two to four spaces

### CODE REVIEW

**Comment by MalwareMechanic** [comment]
@mr-tz linter is saying the version needs to match the fixed version, but when we update a package we always need to increase the version. I propose we update the guidance since it's not clear for metapackages from:

> When a metapackage is named after and installs/configures a specific tool (for instance cygwin.vm that installs/configures cygwin) lock the dependency version (e.g., version="[3.2.0]") and use the locked version as the metapackage's version. If the tool's version uses the 4th segment (e.g., version="[2.0.1.20210213]") update the metapackage's version to use the current date in the 4th segment (i.e., <version>2.0.1.20220224</version>).


to:
> When a metapackage is named after and installs/configures a specific tool (for instance cygwin.vm that installs/configures cygwin) lock the dependency version (e.g., version="[3.2.0]") and use the locked version as the metapackage's version. If the tool's version uses the 4th segment (e.g., version="[2.0.1.20210213]") update the metapackage's version to use the current date (YYYYMMDD) in the 4th segment (i.e., <version>2.0.1.20220224</version>). Additionally, when a change is made to a metapackge, always use the current date in the 4th segment.

Main change is the line:
**Additionally, when a change is made to a metapackge, always use the current date in the 4th segment.**

File: packages/notepadplusplus.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -2,31 +2,45 @@ $ErrorActionPreference = 'Stop'
...
-  }
 
-  if (-Not (Test-Path -Path $configFile)) {
-    VM-Write-Log "WARN" "Can't find config.xml to modify"
-  } else {
-    # Update the config file and disable auto-updates
-    (Get-Content -Path $configFile) -Replace '("noUpdate".*?">)no', '$1yes' | Set-Content -Path $configFile -Force | Out-Null
-  }
+    if (-Not (Test-Path -Path $configFile)) {
+        VM-Write-Log "WARN" "Can't find Notepad++'s config.xml to disable auto-updates, skipping..."
```
Comment (mr-tz) [inline packages/notepadplusplus.vm/tools/chocolateyinstall.ps1:38]: ah, that confused me before (now I know what you meant :D)

**Review by mr-tz — APPROVED** [review]
pending the linter fix

File: packages/notepadplusplus.vm/tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (mr-tz) [reply packages/notepadplusplus.vm/tools/chocolateyinstall.ps1:38]: good change here

**Comment by mr-tz** [comment]
updating the linter now

**Comment by mr-tz** [comment]
linter update addressed in #124

**Pull request merged**